src/app/Components/LanguageInfo.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+  import InputField from "./InputField.svelte";
+  export let properties: any;
+</script>
+
+<style>
+  .language__info {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .language__info-left {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .language__info-right {
+    margin-left: 300px;
+    display: flex;
+    flex-direction: column;
+    order: 2;
+  }
+
+  img {
```

Comment:
**[Suggestion]**: styling with anchor tag is will be a troublesome when it comes to reusing this component

Reply:
@IanSeng could you clarify what you mean by this comment? I'm not sure I understand.

Reply:
let me put my thought into a better sentence hahaha. What I meant was, straight up staying the `img` might not be a best practice.